apps/schematic/api/schematic_api/openapi/openapi.yaml

```diff
@@ -1791,6 +2025,10 @@ components:
       type: object
       x-java-class-annotations:
       - '@lombok.Builder'
+    ComponentLabel:
```

Comment:
For `component/{componentLabel}` endpoint, when I clicked on executing the endpoint, I noticed that it returned: 
```
{
  "detail": "'MolecularEntity' is not a valid component in the schema.",
  "status": 500,
  "title": "Internal error"
}
```
What is the old endpoint called for this one?

Reply:
This was previously `GET /visualize/component`